src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/search.js.template

```diff
@@ -31,8 +31,6 @@ const messages = {
     loading: "##REPLACE:doclet.search.loading##",
     searching: "##REPLACE:doclet.search.searching##",
     redirecting: "##REPLACE:doclet.search.redirecting##",
```

Comment:
Suggestion ...
Would it make sense to "minimize" the use of templates by moving all the template-y stuff to a single template file, that can be included by the other JavaScript files, which would then become "normal" JavaScript `.js` files.

Reply:
That's something we should consider. We could use `script.js` for this as it is the first loaded script file, therefore available in all other script files.

Reply:
I was suggesting that the template-y stuff should be moved to a new template-only file, sort-of like a "template header file", to be used by "pure JavaScript" `*.js` files.